src/libnvme.map

```diff
@@ -6,6 +6,10 @@ LIBNVME_1_3 {
 		nvme_io_mgmt_send;
 		nvme_fdp_reclaim_unit_handle_status;
 		nvme_fdp_reclaim_unit_handle_update;
+		nvme_get_log_fdp_configurations;
+		nvme_get_log_fdp_events;
+		nvme_get_log_fdp_stats;
+		nvme_get_log_reclaim_unit_handle_usage;
```

Comment:
Inline function don't have to be listed in the linker script. While at it, please sort the whole section :)

Reply:
Ah, I see.

In that case, we can drop the commit, or replace it by one removing `nvme_fdp_reclaim_unit_handle_status,update` as well since those are also inline.

Reply:
All good, I got a PR which does the cleanup pending :)